### DISCUSSION TITLE
Option to Hide Movies Completely

### DISCUSSION THREAD

**Discussion by Fmoment1** [trakt/showly#770] [Ideas / Feature Requests]
The app settings provides an option to "disable movies support", but movies still show up as greyed out entries under actors/writers credits. I want to be able to hide movies completely as I only want to use this app for show tracking. Would it be possible? 

P.S: I use this app **without** connecting to Trakt. Nothing is synced.

**Comment by jpsnyder** [comment]
Big +1 on this. I use letterboxed for Movies. So want to ignore movies completely.
It's annoying to always have to do that second click to only show TV Shows when viewing actors.  If I disabled Movies in the settings, any trace of movies shouldn't be present.

**Comment by michaldrabik** [comment]
Will revisit that. Probably makes sense.
